client/components/account-balances/index.tsx

```diff
@@ -130,129 +131,135 @@ const AccountBalances: React.FC = () => {
-							tooltip={
-								<BalanceTooltip
-									label={ `${ fundLabelStrings.pending } tooltip` }
-									content={
-										tab.pendingFunds < 0
-											? interpolateComponents( {
-													mixedString: __(
-														'{{learnMoreLink}}Learn more{{/learnMoreLink}} about why your account balance may be negative.',
-														'woocommerce-payments'
-													),
-													components: {
-														learnMoreLink: (
-															// eslint-disable-next-line jsx-a11y/anchor-has-content
-															<a
-																rel="external noopener noreferrer"
-																target="_blank"
-																href={
-																	documentationUrls.negativeBalance
-																}
-															/>
-														),
-													},
-											  } )
-											: interpolateComponents( {
-													mixedString: sprintf(
-														// Translators: %d is the number of days, e.g. 1, 2, 3, etc.
-														__(
-															'The amount of funds still in the %d day pending period. {{learnMoreLink}}Learn more.{{/learnMoreLink}}',
-															'woocommerce-payments'
-														),
-														tab.delayDays
-													),
-													components: {
-														learnMoreLink: (
-															// eslint-disable-next-line jsx-a11y/anchor-has-content
-															<a
-																rel="external noopener noreferrer"
-																target="_blank"
-																href={
-																	documentationUrls.depositSchedule
-																}
-															/>
-														),
-													},
-											  } )
-									}
-								/>
-							}
-						/>
-					</Flex>
-					{ tab.instantBalance && tab.instantBalance.amount > 0 && (
+			{ ( tab: BalanceTabProps ) => {
+				const totalBalance = tab.availableFunds + tab.pendingFunds;
+
+				const negativeBalanceTooltipContent = interpolateComponents( {
+					mixedString: __(
+						'{{learnMoreLink}}Learn more{{/learnMoreLink}} about why your account balance may be negative.',
+						'woocommerce-payments'
+					),
+					components: {
+						learnMoreLink: (
+							// eslint-disable-next-line jsx-a11y/anchor-has-content
+							<a
+								rel="external noopener noreferrer"
+								target="_blank"
+								href={ documentationUrls.negativeBalance }
+							/>
+						),
+					},
+				} );
+				return (
+					<>
 						<Flex
 							gap={ 0 }
-							className="wcpay-account-balances__instant-deposit"
+							className="wcpay-account-balances__balances"
 						>
-							<InstantDepositButton
-								instantBalance={ tab.instantBalance }
+							<BalanceBlock
+								id={ `wcpay-account-balances-${ tab.currencyCode }-total` }
+								title={ fundLabelStrings.total }
+								amount={ totalBalance }
+								currencyCode={ tab.currencyCode }
+								tooltip={
+									<BalanceTooltip
+										label={ `${ fundLabelStrings.total } tooltip` }
+										maxWidth={
+											totalBalance < 0
+												? undefined
+												: '335px'
```

Comment:
Why are you adding a `maxWidth` prop? This seems overcomplicated, can we set a standard max-width on all tooltips via CSS?

If there is a special need to customise the max width for this tooltip only, I'd suggest using a class + regular CSS to override (since most of our styles use CSS cascade, rather than styles-in-js).

Reply:
> This seems overcomplicated

I agree. Without drilling `maxWidth`, we'll end up with narrower tooltip.

![Screenshot 2024-02-22 at 02 40 25](https://github.com/Automattic/woocommerce-payments/assets/73803630/2439f186-300c-4597-b816-7bf0f2860517)

I actually prefer for more simple diff if we are OK with narrower tooltip like that.

Adding `maxWidth` was necessary because the [BalanceTooltip](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/account-balances/balance-tooltip.tsx#L18) renders [ClickTooltip](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/index.tsx#L109) which renders [TooltipBase](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/tooltip-base.tsx#L174) which [sets an inline `max-width`](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/tooltip-base.tsx#L213) in a function that calculates tooltip position, window dimension etc, so it's pretty complex stuffs to modify.

Reply:
> Adding maxWidth was necessary because the [BalanceTooltip](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/account-balances/balance-tooltip.tsx#L18) renders [ClickTooltip](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/index.tsx#L109) which renders [TooltipBase](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/tooltip-base.tsx#L174) which [sets an inline max-width](https://github.com/Automattic/woocommerce-payments/blob/005f3317b6acea383670af322699de067775be63/client/components/tooltip/tooltip-base.tsx#L213) in a function that calculates tooltip position, window dimension etc, so it's pretty complex stuffs to modify.

I see! That's pretty limiting, I wonder if `TooltipBase` needs to do that. Might be worth logging a (low priority) issue.